client/src/components/content/RobotContainer/RobotContainer.jsx

```diff
@@ -15,16 +16,30 @@ const { Title } = Typography;
  * @category Client
  */
 const RobotContainer = (props) => {
-  const { robotId, robotName } = props;
+  const { robotId, robotName, userId } = props;
   const [name, setRobotName] = useState(robotName);
 
-  const startRobot = () =>
-    alert('Running the Robot is currently not supported!');
+  /**
+   * @description Sends a job to the server to execute a specfic robot for a specific user
+   */
+  const startRobot = () => {
+    // isBotExecutable funtion is to be implemented
+    const isBotExecutable = true;
+    if (isBotExecutable) {
+      socket.emit('robotExecutionJobs', { robotId, userId });
+    } else {
+      alert('Your Bot is not ready to be executed!');
```

Comment:
Maybe a reason would be helpful to display?
```suggestion
      alert('Your Bot is not fully configured and can not be executed!');
```